discovery/pkg/apigee/portalapihandlerjob.go

```diff
@@ -136,12 +136,15 @@ func (j *newPortalAPIHandler) buildServiceBody(newAPI *apigee.APIDocData, produc
 	apiID := fmt.Sprint(newAPI.ID)
 
 	// create attributes to be added to revision and instance
-	serviceAttributes := make(map[string]string)
+	serviceDetails := make(map[string]interface{})
 	revisionAttributes := make(map[string]string)
 	for k, v := range productAttributes {
-		serviceAttributes[k] = v
+		serviceDetails[k] = v
 		revisionAttributes[k] = v
```

Comment:
@sbolosan @jcollins-axway  Should productAttributes get merged into the x-agent-details for both the APIService and APIRevision?